spec/spec_helper.cr

```diff
@@ -61,15 +61,22 @@ end
 
 Spec.before_each do
   Jennifer::Adapter.adapter.begin_transaction
+  reset_config
```

Comment:
Why we do this twice (before and after)

Reply:
The config tests manipulate the config state which could bleed into other tests. The second call in after was just to be ultra sure that the system is always consistent, in a known configuration state before the test and reset to a known config state after. I guess as there's only one set of hooks the before hook might be enough as its run before each test.